packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -771,7 +816,11 @@ export default decorate([
                         <Ul>
                           {map(state.remotes, (id, key) => (
                             <Li key={id}>
-                              <Remote id={id} />
+                              <Remote id={id} />{' '}
+                              <RemoteProxyWarning
+                                id={id}
+                                proxyId={state.proxyId}
+                              />
```

Comment:
Why don't you handle the warning condition here instead of adding n subscriptions to the `remotes`?

Reply:
Because it will be mutualized between `backup NG` and `Metadata Backup` and it not cost resources because it just access to the cash.